lib/saml/user_attributes/base.rb

```diff
@@ -16,9 +16,17 @@ def initialize(saml_attributes, authn_context)
       # Common Attributes
       # ID.me unique identifier
       def uuid
+        idme_uuid
+      end
+
+      def idme_uuid
         attributes['uuid']
       end
 
+      def sec_id
```

Comment:
Any user_attributes derived from Base (which really means "the variants of SAML payloads we get direct from ID.me) will never have a sec_id attribute (because ID.me will never have it, nor will either of the federated credential types). So I think this just ought to return nil.